File: JobAggregatorFront/src/app/models/DTOs/organization-usersDTO.ts
import { Handbook } from "../handbook";

export interface OrganizationUsersDTO{
    id: number;
    name: string;
    description: string;
    website?: string;
    email: string;
    phoneNumber: number;
    address?: string;
    activities?: Handbook[];
    idUsers: number[]
}

File: JobAggregator/JobAggregatorApi/DTO/OrganizationUsersDTO.cs
using JobAggregator.Core.Entities;

namespace JobAggregator.Api.DTO;

public class OrganizationUsersDTO
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public string? Website { get; set; }
    public string Email { get; set; }
    public int? PhoneNumber { get; set; }
    public string? Address { get; set; }
    public List<int> IdUsers { get; set; } = [];
    public List<Activity>? Activities { get; set; }
}
